eng/common/scripts/Package-Properties.ps1

```diff
@@ -188,7 +188,7 @@ function Get-PrPkgProperties([string]$InputDiffJson) {
     }
 
     foreach ($addition in $additionalValidationPackages) {
-        $key = $addition.Replace($RepoRoot, "").TrimStart('\/')
+        $key = $addition.Replace($RepoRoot, "").TrimStart('\/').Replace("\", "/")
```

Comment:
Should the replace happen before the trim?

Reply:
I don't think it really matters. If we trimStart, we're reducing the work to replace "\" with "/" by one character. That's about the only impact though.